### DISCUSSION TITLE
Keyboard inputs are glitching!

### DISCUSSION THREAD

**Discussion by DANIHEX** [coder/code-server#5777] [Q&A]
Hi, I don't know if this is happening only to me but when I press Tab key it doesn’t insert 4 spaces it selects status bar strangely or when I press Enter key it don't auto complete, it just inserts new line even if it set to auto complete in settings.
Looks like the whole keyboard is glitching or maybe code-server is reading keyboard inputs incorrectly(In settings keyboard short cuts when enabling read input, it doesn’t recognize Tab and Enter, it says “unknown”).
Is this a bug? Or how can I fix this?
Thanks!

Discord: Hexiter#0001

Platform: Android, Termux, Ubuntu CLI
Browser: Opera Android
Device: SAMSUNG Galaxy A30 - 32 GB MEMORY/3 GB RAM

**Comment by jsjoeio** [comment]
Hi! Hmm...can you reproduce on the same device using vscode.dev?

**Comment by duangan1** [comment]
你好，我是ysy，我已收到你的邮件，会尽快查看的，谢谢！
